packages/terra-application-utility/src/ApplicationHeaderUtility.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import Utils from './_Utils';
+import UtiltyButton from './utility/_UtilityButton';
+
+const propTypes = {
+  /**
+   * The array containing the menu items to be rendered within the menu.
```

Comment:
I don't know if this provides enough information of what shape the items in the array should be to the user.

Reply:
One idea to provide users more context of `Utils.itemShape` is to set a default prop for menuItems which matches the shape of `Utils.itemShape`. This should show up in the prop-table doc.

Reply:
Per offline discussion with @bjankord, we'll add a menu item section to the readme.

Reply:
[Added](https://github.com/cerner/terra-framework/pull/103/commits/efd64c3eed5f7146aaefa0505339e34e92f75bed#diff-c2a019a5d2aa1bb5a4f83063a9a651e0R10) new menu section.